.travis.yml

```diff
@@ -4,9 +4,13 @@ node_js:
   - "4.3"
 
 before_install:
-  - "export CHROME_BIN=chromium-browser"
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
+  - "sudo apt-get update"
```

Comment:
Agreed, not sure why this is necessary. I got it from http://blog.500tech.com/setting-up-travis-ci-to-run-tests-on-latest-google-chrome-version/